pages/new-page.md

```diff
@@ -1,7 +1,7 @@
 ---
 permalink: /adding-a-new-page/
 layout: default
-title: Adding a new page
+title: Adding a New Page
```

Comment:
Ah ok. FYI I noticed that `{% if page.title == link.text %}` check didn't work if the case didn't match.